app/assets/stylesheets/govuk_publishing_components/components/_table.scss

```diff
@@ -16,6 +16,19 @@ $table-row-even-background-colour: govuk-colour("light-grey", $legacy: "grey-4")
   color: $govuk-secondary-text-colour;
 }
 
+.govuk-table-filter {
+  display: none;
+}
+
+.govuk-table__message {
+  @include govuk-font($size: 19);
+}
+
+.govuk-table__row--hidden,
+.govuk-table__message--hidden {
+  display: none;
```

Comment:
I suspect that there's already a class in `govuk-frontend` available to us that will hide things. Might be worth asking in the Design System slack channel, might save you a few lines of CSS here.

Reply:
Great, thanks, I've used that in place of `display: none` here.